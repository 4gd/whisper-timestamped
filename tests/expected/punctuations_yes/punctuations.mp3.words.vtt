WEBVTT

00:00.380 --> 00:01.300
Dis-moi,

00:01.300 --> 00:01.460
est-ce

00:01.460 --> 00:01.640
que

00:01.640 --> 00:02.000
l'avion

00:02.000 --> 00:02.400
vole?

