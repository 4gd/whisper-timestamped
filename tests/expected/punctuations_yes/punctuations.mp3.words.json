{
  "text": " Dis-moi, est-ce que l'avion vole?",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.38,
      "end": 2.4,
      "text": " Dis-moi, est-ce que l'avion vole?",
      "tokens": [
        50364,
        4208,
        12,
        29292,
        11,
        871,
        12,
        384,
        631,
        287,
        6,
        706,
        313,
        49877,
        2506,
        50494
      ],
      "temperature": 0.0,
      "avg_logprob": -0.30108165740966797,
      "compression_ratio": 0.8048780487804879,
      "no_speech_prob": 0.05284806713461876,
      "confidence": 0.926,
      "words": [
        {
          "text": "Dis-moi,",
          "start": 0.38,
          "end": 1.3,
          "confidence": 0.803
        },
        {
          "text": "est-ce",
          "start": 1.3,
          "end": 1.46,
          "confidence": 0.966
        },
        {
          "text": "que",
          "start": 1.46,
          "end": 1.64,
          "confidence": 0.977
        },
        {
          "text": "l'avion",
          "start": 1.64,
          "end": 2.0,
          "confidence": 0.994
        },
        {
          "text": "vole?",
          "start": 2.0,
          "end": 2.4,
          "confidence": 0.891
        }
      ]
    }
  ],
  "language": "fr"
}